bin/heat-flow-module-L0/9_plate_cooling_model_of_lithosphere.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def fourier_term (n, t):
</pre><p>A general comment - do we want to follow python best practices here and add proper documentation for each function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Good point!I will add the documentation of all functions.